src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -4517,6 +4657,33 @@ public class AddressAdminClient implements BackgroundResource {
    * }
    * </code></pre>
    *
+   * @param region Name of the region for this request.
+   * @param address A reserved address resource.
+   * @throws com.google.api.gax.rpc.ApiException if the remote call fails
+   */
+  @BetaApi
+  public final Operation addressesInsert(RegionName region, Address address) {
```

Comment:
Comment for later work: to match our existing API naming patterns, this should be `insertAddresses`